docs/docs/api-reference.md

```diff
@@ -270,6 +274,10 @@ Opens invite modal inside the Rune app. Useful if you want to incentivize player
 
 Returns the amount of milliseconds that have passed since the start of the game. See [Using Time in your Game](advanced/real-time-games.md#game-time).
 
+### `Rune.worldTime()` {#runeworldtime}
+
+Returns the amount of milliseconds since the start of epoch. See [Using Time in your Game](advanced/real-time-games.md#world-time).
```

Comment:
```suggestion
Returns the amount of milliseconds since the start of epoch with a precision of 1 second. See [Using Time in your Game](advanced/real-time-games.md#world-time).
```